openlibrary/catalog/marc/tests/test_data/bin_expect/zweibchersatir01horauoft_meta.json

```
@@ -18,17 +18,25 @@
   "publish_country": "ge",
   "authors": [
     {
-      "personal_name": "Horace",
       "name": "Horace",
       "entity_type": "person"
+    },
+    {
+      "name": "Kirchner, Carl Christian Jacob",
+      "birth_date": "1787",
+      "death_date": "1855",
+      "entity_type": "person",
+      "role": "tr. [and] ed"
```

```suggestion
      "role": "tr. [and] ed."
```
Keep trailing period